<?xml version="1.0" encoding="UTF-8"?><testrun name="CommentServiceTest" project="backend" tests="7" started="7" failures="3" errors="3" ignored="0">
  <testsuite name="com.novelsbr.backend.services.CommentServiceTest" time="2.714" displayname="CommentServiceTest" uniqueid="[engine:junit-jupiter]/[class:com.novelsbr.backend.services.CommentServiceTest]">
    <testcase name="preparingTestEnvironment" classname="com.novelsbr.backend.services.CommentServiceTest" time="2.209" displayname="preparingTestEnvironment()" uniqueid="[engine:junit-jupiter]/[class:com.novelsbr.backend.services.CommentServiceTest]/[method:preparingTestEnvironment()]"/>
    <testcase name="save" classname="com.novelsbr.backend.services.CommentServiceTest" time="0.118" displayname="save()" uniqueid="[engine:junit-jupiter]/[class:com.novelsbr.backend.services.CommentServiceTest]/[method:save()]">
      <error>java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0&#13;
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)&#13;
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)&#13;
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)&#13;
	at java.base/java.util.Objects.checkIndex(Objects.java:359)&#13;
	at java.base/java.util.ArrayList.get(ArrayList.java:427)&#13;
	at com.novelsbr.backend.services.CommentServiceTest.save(CommentServiceTest.java:105)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;
</error>
    </testcase>
    <testcase name="findAll" classname="com.novelsbr.backend.services.CommentServiceTest" time="0.096" displayname="findAll()" uniqueid="[engine:junit-jupiter]/[class:com.novelsbr.backend.services.CommentServiceTest]/[method:findAll()]">
      <failure>
        <expected>false</expected>
        <actual>true</actual>
        org.opentest4j.AssertionFailedError: expected: &lt;false&gt; but was: &lt;true&gt;&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#13;
	at org.junit.jupiter.api.AssertFalse.failNotFalse(AssertFalse.java:63)&#13;
	at org.junit.jupiter.api.AssertFalse.assertFalse(AssertFalse.java:36)&#13;
	at org.junit.jupiter.api.AssertFalse.assertFalse(AssertFalse.java:31)&#13;
	at org.junit.jupiter.api.Assertions.assertFalse(Assertions.java:231)&#13;
	at com.novelsbr.backend.services.CommentServiceTest.findAll(CommentServiceTest.java:128)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;

      </failure>
    </testcase>
    <testcase name="findAllNovelsByEntityId" classname="com.novelsbr.backend.services.CommentServiceTest" time="0.09" displayname="findAllNovelsByEntityId()" uniqueid="[engine:junit-jupiter]/[class:com.novelsbr.backend.services.CommentServiceTest]/[method:findAllNovelsByEntityId()]">
      <failure>
        <expected>1</expected>
        <actual>0</actual>
        org.opentest4j.AssertionFailedError: expected: &lt;1&gt; but was: &lt;0&gt;&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#13;
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:531)&#13;
	at com.novelsbr.backend.services.CommentServiceTest.findAllNovelsByEntityId(CommentServiceTest.java:141)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;

      </failure>
    </testcase>
    <testcase name="findAllChaptersByEntityId" classname="com.novelsbr.backend.services.CommentServiceTest" time="0.08" displayname="findAllChaptersByEntityId()" uniqueid="[engine:junit-jupiter]/[class:com.novelsbr.backend.services.CommentServiceTest]/[method:findAllChaptersByEntityId()]">
      <failure>
        <expected>1</expected>
        <actual>0</actual>
        org.opentest4j.AssertionFailedError: expected: &lt;1&gt; but was: &lt;0&gt;&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#13;
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:531)&#13;
	at com.novelsbr.backend.services.CommentServiceTest.findAllChaptersByEntityId(CommentServiceTest.java:152)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;

      </failure>
    </testcase>
    <testcase name="update" classname="com.novelsbr.backend.services.CommentServiceTest" time="0.057" displayname="update()" uniqueid="[engine:junit-jupiter]/[class:com.novelsbr.backend.services.CommentServiceTest]/[method:update()]">
      <error>java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0&#13;
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)&#13;
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)&#13;
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)&#13;
	at java.base/java.util.Objects.checkIndex(Objects.java:359)&#13;
	at java.base/java.util.ArrayList.get(ArrayList.java:427)&#13;
	at com.novelsbr.backend.services.CommentServiceTest.update(CommentServiceTest.java:159)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;
</error>
    </testcase>
    <testcase name="delete" classname="com.novelsbr.backend.services.CommentServiceTest" time="0.028" displayname="delete()" uniqueid="[engine:junit-jupiter]/[class:com.novelsbr.backend.services.CommentServiceTest]/[method:delete()]">
      <error>java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0&#13;
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)&#13;
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)&#13;
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)&#13;
	at java.base/java.util.Objects.checkIndex(Objects.java:359)&#13;
	at java.base/java.util.ArrayList.get(ArrayList.java:427)&#13;
	at com.novelsbr.backend.services.CommentServiceTest.delete(CommentServiceTest.java:172)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
